iree/turbine/kernel/wave/iree_utils.py

```diff
@@ -93,7 +94,7 @@ def get_mmt_asm(
     if not cast_fp8:
         matmul_function = f"""
         func.func @{func_name}(%lhs: tensor<{lhs_type}>, %rhs: tensor<{rhs_type}>) -> tensor<{acc_type}> {{
-          %c0 = arith.constant 0.0 : {acc_dtype}
+          %c0 = arith.constant {zero} : {acc_dtype}
```

Comment:
nit: you probably can just do `zero = "0.0" if acc_dtype.starts_with("f") else "0"` instead of passing it as parameter.

Reply:
nice idea, done!